src/aap_eda/settings/default.py

```diff
@@ -617,6 +617,13 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED = settings.get(
     "DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED", False
 )
+if DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
+    RQ_PERIODIC_JOBS.append(
+        {
+            "func": "aap_eda.tasks.shared_resources.resync_shared_resources",
+            "interval": 900,
```

Comment:
@bzwei Should this be slightly lower than 15 minutes so we don't overlap when the new one is about to start. Also how do we specify the job timeout?

Reply:
I am working on adding a lock to SyncExecutor that will not allow 2 instances to execute at the same time.